docs/apis-clients/console-api-reference.md

```diff
@@ -1,37 +1,39 @@
 ---
 id: console-api-reference
 title: Console API clients (REST)
-description: "Create and manage clusters, and interact with Camunda Platform 8 programmatically without using the Camunda Platform 8 UI."
+description: "Create and manage clusters, and interact with Camunda Platform 8 programmatically without using the Camunda Platform 8 Console."
 ---
 
 ## Console API (REST)
 
-For all requests, include the access token for Cloud API in the Authorization header: `authorization:Bearer ${TOKEN}`.
+For all requests, include the access token in the Authorization header: `authorization:Bearer ${TOKEN}`.
```

Comment:
Nice catch on Authorization header -- so bizarre that Authorization is capitalized, but it is technically correct 🤷‍♀️